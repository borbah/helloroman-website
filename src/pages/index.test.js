import React from 'react';
import IndexPage from './';

describe('Index page', () => {
  it('should render', () => {
    const wrapper = shallow(<IndexPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
