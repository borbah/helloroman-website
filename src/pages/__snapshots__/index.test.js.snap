// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render 1`] = `
<div>
  <h1>
    Hello!
  </h1>
  <p>
    Iâ€™m Roman and this is my personal website
  </p>
</div>
`;
